edd-license-free-download.php

```diff
@@ -155,7 +155,7 @@ public static function save_data( $post_id ) {
 		if ( ! empty( $_POST['edd_lfd_products'] ) && is_array( $_POST['edd_lfd_products'] ) ) {
 			foreach ( $_POST['edd_lfd_products'] as $product ) {
 				if ( ! empty( $product ) ) {
-					$products[] = absint( $product );
+					$products[] = esc_attr( $product );

```

Comment:
Maybe `sanitize_text_field()` is better here?

Typically you want `sanitize_{x}` when _saving data_ and `esc_{x}` when _displaying_ data. We're saving here, so a sanitize option is better.